### DISCUSSION TITLE
How to make €/l not end up in L^-1*€

### DISCUSSION THREAD

**Discussion by ManuelSchneid3r** [Qalculate/libqalculate#510] [Q&A] [answered]
When using the lib by default the simple 1€/L expression evaluates to 1L^-1*€. I checked and tried nearly every combination of eval and print opts. How can I avoid this?

**Comment by hanna-kn** [comment, marked as answer]
Call MathStructure::format() (with print opts) before MathStructure::print(), or use Calculator::print() or Calculator::calculateAndPrint().
